src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,50 @@
+package command;
+
+import java.io.IOException;
+
+import task.Task;
+import util.Storage;
+import util.TaskList;
+import util.Ui;
+
+
+/**
+ * Represents the delete command. The delete command removes a task from the task list.
+ */
+public class DeleteCommand extends Command {
+    /**
+     * Integer representing the task number to be deleted.
+     */
+    private final int taskNum;
+
+    /**
+     * Creates a new Delete command.
+     *
+     * @param taskNum Task number of task to be deleted.
+     */
+    public DeleteCommand(int taskNum) {
+        this.taskNum = taskNum;
+    }
+
+    /**
+     * Executes the delete command. The execution involves deleting the task from the task list,
+     * writing to the storage as well as printing the relevant UI.
+     *
+     * @param lst     List containing the current tasks.
+     * @param ui      Ui allows execute to carry out ui methods to print to the console.
+     * @param storage Storage allows execute to write and read files.
+     */
+    public String execute(TaskList lst, Ui ui, Storage storage) {
```

Comment:
Missing return tag in javadocs

"Omit @return for methods that return void and for constructors; include it for all other methods, even if its content is entirely redundant with the method description. Having an explicit @return tag makes it easier for someone to find the return value quickly. Whenever possible, supply return values for special cases (such as specifying the value returned when an out-of-bounds argument is supplied)."

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@return

Reply:
Good catch, missed this one out will add the return tag.